src/prints.jl

```diff
@@ -0,0 +1,44 @@
+using Printf
```

Comment:
move this to `LinearRegression.jl` and add `Printf`as a dependency

Reply:
Im trying to add Printf as a dependency using `] add Printf`, in the environment of the project. However, the julia terminal says that there are no changes in the Project.toml, I've already checked and thats true. Maybe its happening because Printf is a package that already comes with the julia distribution, what do you think about it?